workflow/landing-page/index.html

```diff
@@ -182,171 +182,10 @@ <h3>Learning New Things</h3>
-              <span class="blue-color">template</span><span class="green-color">mo</span>. by template There are total
-              12 HTML pages. Credit goes to <a class="orange-color" href="https://unsplash.com/">Unsplash</a> for
-              images.
-            </p>
-          </div>
-        </div>
-        <div class="white-background-color image-block">
-          <div class="image-block-photo-wrapper">
-            <img class="image-block-photo" src="assets/blog7.png" />
-            <div class="overlay-image">
-              <div class="button button-image">
-                <i class="fas fa-link"></i>
-              </div>
-            </div>
-          </div>
-          <div class="image-block-information">
-            <h3 class="image-block-photo-name">An Open Letter to Larry Page</h3>
-            <p class="image-block-date roboto-font">6 November 2084 by Michael</p>
-            <p class="primary-text roboto-font">
-              Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit tempora veritatis distinctio similique?
-              Quis quasi dolor nihil, perspiciatis officia facere excepturi?
-            </p>
-          </div>
-        </div>
-        <div class="white-background-color image-block">
-          <div class="image-block-photo-wrapper">
-            <img class="image-block-photo" src="assets/blog2.png" />
-            <div class="overlay-image">
-              <div class="button button-image">
-                <i class="fas fa-link"></i>
-              </div>
-            </div>
-          </div>
-          <div class="image-block-information">
-            <h3 class="image-block-photo-name">A Love Letter to the City</h3>
-            <p class="image-block-date roboto-font">2 November 2084 by Michael</p>
-            <p class="primary-text roboto-font">
-              Lorem ipsum dolor sit amet consectetur adipisicing elit. Vero perferendis tenetur nesciunt numquam.
-              Excepturi aperiam eligendi maiores.
-            </p>
-          </div>
-        </div>
-        <div class="white-background-color image-block">
-          <div class="image-block-photo-wrapper">
-            <img class="image-block-photo" src="assets/blog6.png" />
-            <div class="overlay-image">
-              <div class="button button-image">
-                <i class="fas fa-link"></i>
-              </div>
-            </div>
-          </div>
-          <div class="image-block-information">
-            <h3 class="image-block-photo-name">Porto Design Summer School</h3>
-            <p class="image-block-date roboto-font">4 November 2084 by Christina</p>
-            <p class="primary-text roboto-font">
-              Lorem ipsum dolor sit amet, consectetur adipisicing elit. Alias, deleniti, id quibusdam aut optio saepe
-              soluta tempore neque voluptatum.
-            </p>
-          </div>
-        </div>
+      <div class="image-block-wrapper"></div>
```

Comment:
as it's wrappers for all cards (not for one), it's better to name it `images-blocks-wrapper`